admin/class-admin.php

```diff
@@ -577,6 +577,7 @@ public function filter_stopwords_from_slug( $slug, $post_title ) {
 	 * @return array
 	 */
 	public function filter_settings_pages( array $pages ) {
+		$this->turn_on_advanced_settings();
```

Comment:
This violates the single responsibility principle. I would be better to do it outside this filter function.